### ISSUE TITLE
IPT testing - Camtrap DP resource from an archive - not all seems auto-mapped

### ISSUE THREAD

**Issue by ManonGros** [gbif/ipt#2303]
I am using a local version of the IPT 3 [org/gbif/ipt/3.0-RC2-SNAPSHOT/ipt-3.0-RC2-20240124.165554-116.war](https://repository.gbif.org/repository/gbif/org/gbif/ipt/3.0-RC2-SNAPSHOT/ipt-3.0-RC2-20240124.165554-116.war)

I uploaded this archived https://download-directory.github.io/?url=https://github.com/tdwg/camtrap-dp/tree/1.0/example to the IPT. And the observationIDs didn't seem to be found (even though the header and mapping seems correct?) The media license was also blank.

See the screenshot for the observationID.
<img width="1351" alt="Screenshot 2024-01-25 at 11 59 28" src="https://github.com/gbif/ipt/assets/7677271/3182598c-f922-42ec-bea3-a9c975248d02">
<img width="1349" alt="Screenshot 2024-01-25 at 12 01 13" src="https://github.com/gbif/ipt/assets/7677271/d1dc5ea4-151c-46fd-bc7e-bebcc3dd91fd">

issue for testing here: https://github.com/gbif/ipt/issues/2283

**Comment by mike-podolskiy90** [comment]
Thanks Marie

I believe it's because of the BOM (byte order mark) character in the file.

**Comment by peterdesmet** [comment]
Indeed, could you try https://download-directory.github.io/?url=https://github.com/tdwg/camtrap-dp/tree/188d285a3765f379f7c16ac2f542090cc1dfc86a/example (the latest commit on Camtrap DP where the BOM issue should be resolved)?

**Comment by mike-podolskiy90** [comment]
Yes, it's fine

**Comment by ManonGros** [comment]
Thanks! Yes the header is mapped but there is still no media license:

<img width="1026" alt="Screenshot 2024-01-25 at 13 24 55" src="https://github.com/gbif/ipt/assets/7677271/80ad9475-e766-4bd0-bbc7-5a43a386a6d5">

I think there should be, right?
<img width="657" alt="Screenshot 2024-01-25 at 13 25 31" src="https://github.com/gbif/ipt/assets/7677271/0ee5419d-ee70-4cb9-80ae-38fdc2a9987f">

**Comment by mike-podolskiy90** [comment]
Yes, I'll be fixing that in the issue Andrea reported earlier

**Comment by mike-podolskiy90** [comment]
I think we need to require `name` field for the license. We need to display that on the UI. Otherwise we'll have to infer that from the path which I don't like
